okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/utf8.kt

```diff
@@ -23,7 +23,8 @@ import okio.Buffer
  * sample of code points to detect unicode control characters commonly used in binary file
  * signatures.
  */
-internal fun Buffer.isProbablyUtf8(): Boolean {
+// TODO make non public
+fun Buffer.isProbablyUtf8(): Boolean {
```

Comment:
Breaks because of

```
main.java.srcDirs += project.file("../okhttp-logging-interceptor/src/test/java")
```

Reply:
Yeah, we can just move this to okhttp3.logging.internal?